gen_cert.py

```diff
@@ -81,10 +81,13 @@ def get_cert_date(
     """
     Get pertinent date for display on cert
     """
-    if force_date:
-        date_value = format_datetime(force_date, 'MMMM d, y', tzinfo=timezone, locale=locale)
-    else:
-        date_value = format_datetime(datetime.datetime.today(), 'MMMM d, y', tzinfo=timezone, locale=locale)
+    utc_zone = get_timezone(settings.TIMEZONE)
```

Comment:
This name is no longer accurate. `from_zone` matches the original POC and follows the existing convention, `to_zone`.